src/screen.js

```diff
@@ -0,0 +1,65 @@
+
+import config from './config'
+
+const AlpineScreenMagicMethod = {
+    start() {
+        let initialized
+
+        Alpine.addMagicProperty('screen', function ($el) {
+            // bind resize event to window with debounce
```

Comment:
Yeah, we can change it. btw you sure all other helpers use arrow function? I saw few doesn't.

Reply:
Just checked now and they all seem to be. Maybe you're thinking of the return function? Those appear to be mixed (and should probably be matched up in a PR too)

Reply:
Hmm, i can't see a clear path, where i also copy one of the existing helper to start with so i just followed. So can i just use arrow functions instead of normal function anywhere i need to use?

Reply:
Oh, i think my fork was older version where it was still used normal functions, after your merge the object pr, i see it.